testprojects/src/java/org/pantsbuild/testproject/build_file_imports_function/hello.py

```diff
@@ -0,0 +1,3 @@
+from __future__ import print_function
+
+print("Hello")
```

Comment:
Do we have a more general and vague rule, like `genrule`, rather than me needing to arbitrarily pick python as a language to write hello world in? Because really, this is very language agnostic...

But either way, moved.